editor/src/core/shared/redux-devtools.ts

```diff
@@ -81,12 +77,30 @@ function simplifiedMetadataMap(metadata: ElementInstanceMetadataMap) {
   return sanitizedSpyData
 }
 
+function sanitizeEditor(editor: EditorState) {
+  // When you debug something, feel free to add it to the sanitized editor. right now it contains a few keys that I needed.
+  // Be careful about what you add: logging too much stuff chokes the redux devtool
+  return {
+    selectedViews: editor.selectedViews.map(EP.toString) as any, // this is easier for human consumption
+    canvas: {
+      mountCount: editor.canvas.mountCount,
+      domWalkerInvalidateCount: editor.canvas.domWalkerInvalidateCount,
+      canvasContentInvalidateCount: editor.canvas.canvasContentInvalidateCount,
+      interactionSession: {
+        metadata: simplifiedMetadataMap(editor.canvas.interactionSession?.metadata ?? {}) as any,
+      },
+    } as Partial<EditorState['canvas']>,
```

Comment:
```suggestion
    } as Partial<EditorStateCanvas>,
```
As there's now a type for that field.